CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 The change log describes what is "Added", "Removed", "Changed" or "Fixed" between each release.
 
+## 0.2.0
```

Comment:
Should this be 0.4.0 or 1.0.0 to fix the BC break issue?

Reply:
0.4.0